docs/manual/introduction.tex

```diff
@@ -557,6 +557,8 @@
   which is a javac argument to halt compilation if a warning is issued.
 \item \<-AignoreInvalidAnnotationLocations>
   Ignore annotations in bytecode that have invalid annotation locations.
+\item \<-AnoWarnOnTypeCheckingHalt>
+  Don't issue the ``Javac errored; type checking halted.'' warning.
```

Comment:
Does type-checking still halt in this case?  (I assume so, but please make it explicit either way.)

When would a user want to pass this command-line argument?  If we don't have a compelling use-case, maybe this flag should not be added.

I see that two tests use it and they fail if it is removed, but I'm not sure why.
For instance, one fails this way:

```
org.checkerframework.framework.test.junit.InitializedFieldsValueTest > run[/home/mernst/research/types/checker-framework-fork-smillst-branch-no-null-element/framework/initialized-fields-value] FAILED
    java.lang.AssertionError: 3 out of 3 expected diagnostics were found.
    1 unexpected diagnostic was found:
    :0: warning: Javac errored; type checking halted.
    While type-checking /home/mernst/research/types/checker-framework-fork-smillst-branch-no-null-element/framework/tests/initialized-fields-value/ClassInitializer.java, /home/mernst/research/types/checker-framework-fork-smillst-branch-no-null-element/framework/tests/initialized-fields-value/ClassInitializer2.java, /home/mernst/research/types/checker-framework-fork-smillst-branch-no-null-element/framework/tests/initialized-fields-value/ClassInitializer2a.java, /home/mernst/research/types/checker-framework-fork-smillst-branch-no-null-element/framework/tests/initialized-fields-value/ClassInitializer3.java, /home/mernst/research/types/checker-framework-fork-smillst-branch-no-null-element/framework/tests/initialized-fields-value/ManualConstructor.java
```

but I am able to run `javac` on all of those files without problem.